web/styles/dark_theme.css

```diff
@@ -720,8 +734,11 @@
         }
     }
 
-    & li.active-filter,
-    li.active-sub-filter {
+    & li.active-filter {
+        background-color: var(--color-background);
+    }
```

Comment:
I'm not sure if this line is what's responsible, but the behavior of hovering over the current stream narrow in the left sidebar is weird in a way that looks like a selector priority bug -- in the dark theme there's no change in appearance on hover at all.

Reply:
thanks for pointing this out -- fixed now!


https://user-images.githubusercontent.com/5634097/231871539-bd809934-896b-4f05-ba79-16bdc6a011e0.mp4


https://user-images.githubusercontent.com/5634097/231871567-8e83abe2-4afd-4499-8447-ca1663575e9d.mp4